### ISSUE TITLE
[feature req] Dump out execution analysis in JSON 

### ISSUE THREAD

**Issue by lukebyrne** [hasura/graphql-engine#2424]
Hi guys, 

I use this pretty cool tool to try to optimize querys

http://tatiyants.com/pev/#/plans/new

Would be great if the query plan dumped out the execution analysis in JSON as well so it can be easily pasted into the tool.

Cheers,

Luke

**Comment by dariocravero** [comment]
I agree with @lukebyrne, alternatively, we could integrate it within the console. Here's a remake of that project that could be considered too https://github.com/dalibo/pev2
